src/main/java/com/hwr/hwrzeiterfassung/database/controller/TimeEntryController.java

```diff
@@ -0,0 +1,61 @@
+package com.hwr.hwrzeiterfassung.database.controller;
+
+import com.hwr.hwrzeiterfassung.database.repositorys.TimeRepository;
+import com.hwr.hwrzeiterfassung.database.tables.Day;
+import com.hwr.hwrzeiterfassung.database.tables.Project;
+import com.hwr.hwrzeiterfassung.database.tables.Time;
+import org.joda.time.DateTime;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.NoSuchElementException;
+
+@Controller
+@RequestMapping("/timeEntry")
+public class TimeEntryController {
+    @Autowired
+    private TimeRepository timeRepository;
+
+    @Autowired
+    private LoginController loginController;
+    @Autowired
+    private DayController dayController;
+    @Autowired
+    private ProjectController projectController;
+
+    @PostMapping("/addTimeEntry")
+    public ResponseEntity<HttpStatus> addTimeEntry(
+            @RequestParam String email, @RequestParam String password,
+            @RequestParam long dayId,
+            @RequestParam @DateTimeFormat(pattern = "yyyy-MM-dd HH:mm:ss") DateTime start,
+            @RequestParam @DateTimeFormat(pattern = "yyyy-MM-dd HH:mm:ss") DateTime end,
+            @RequestParam boolean isBreak, @RequestParam String note, @RequestParam int projectId){
+
+        if (!loginController.validateLoginInformation(email, password))
+            throw new ResponseStatusException(HttpStatus.NOT_ACCEPTABLE, "User Credentials invalid");
+
+        try{
+            Day day = dayController.getDayById(dayId);
+            Project project = projectController.getProjectById(projectId);
+
+            Time time = new Time();
```

Comment:
hier kann man einfach einen construktor benutzen

Reply:
+1 
Der entsprechende Konstruktor müsste dann halt noch in Time hinzugefügt werden. 
(Da gibt's aktuell nur NoArgs)

Reply:
Dann müsste man aber in der Time-Klasse `@NoArgsConstructor` durch `@AllArgsConstructor` ersetzen. Mache ich gleich.